internal/fxreflect/fxreflect.go

```diff
@@ -23,12 +23,15 @@ package fxreflect
 import (
 	"fmt"
 	"reflect"
+	"regexp"
 	"runtime"
 	"strings"
 
 	"go.uber.org/dig"
 )
 
+var vendorRe = regexp.MustCompile("^.*?vendor/")
```

Comment:
A comment here for regexp noobs would be useful, e.g. "match the first occurring vendor/ in  the file path".